docs/assets/scripts/hedwig-main.umd.js

```diff
@@ -6574,6 +6581,45 @@
     }
     customElements.define('cpu-user-usage', UserUsage);
 
+    /**
+     * @name CpuCount
+     * @description
+     * @extends HTMLElement
+     * Graph representing CPU Count
+     */
+
+    class CpuCount extends HTMLElement {
+      constructor() {
+        super();
+      }
+      /**
+       * @name connectedCallback
+       * @description
+       * Call back for when the component is attached to the DOM
+       */
+
+
+      connectedCallback() {
+        var defaults = new Defaults();
+        var margin = this.dataset.margin || defaults.margin;
+        var height = (this.dataset.height || defaults.height) - margin.top - margin.bottom;
+        var width = (this.dataset.width || defaults.width) - margin.left - margin.right;
+        var lineColor = this.dataset.lineColor || defaults.lineColor;
+        var unit = this.dataset.unit || 'count';
```

Comment:
Thinking we can have some mechanism in place to change the unit of measurement from percentage.

Reply:
yea I have been thinking the same thing. I think we can use data-* attrs for that